airbyte-integrations/connectors/destination-sftp-json/README.md

```diff
@@ -1,118 +1,91 @@
-# Sftp Json Destination
+# Sftp-Json source connector
```

Comment:
Hmmmm, I think you'd have to check out the old readme and manually change it. I think it also could be that we messed up other connectors like that too.